src/ocf_seq_cutoff.c

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
How about writing some kind of short description of this feature? I think commit message would be good place for it.